include/kvdk/status.h

```diff
@@ -21,23 +21,20 @@
 extern "C" {
 #endif
 
-#define FOREACH_ENUM(GEN)                                                      \
-  GEN(Ok)                                                                      \
-  GEN(NotFound)                                                                \
-  GEN(MemoryOverflow)                                                          \
-  GEN(PmemOverflow)                                                            \
-  GEN(NotSupported)                                                            \
-  GEN(MapError)                                                                \
-  GEN(BatchOverflow)                                                           \
-  GEN(TooManyWriteThreads)                                                     \
-  GEN(InvalidDataSize)                                                         \
-  GEN(IOError)                                                                 \
-  GEN(InvalidConfiguration)                                                    \
-  GEN(Abort)
-#define GENERATE_ENUM(ENUM) ENUM,
-#define GENERATE_STRING(STRING) #STRING,
-
-typedef enum { FOREACH_ENUM(GENERATE_ENUM) } KVDKStatus;
+typedef enum {
+  Ok,
```

Comment:
please roll back the previous code, because `GENERATE_STRING` can convert enum to string. User can use it easily. Thanks a lot

Reply:
done